drivers/medicaid_hmis_interchange/app/models/medicaid_hmis_interchange/health/submission.rb

```diff
@@ -54,16 +54,25 @@ def response_filename
       @timestamp.strftime(TIMESTAMP_FORMAT)
     end
 
-    # Clients are included in the submission if they are enrolled in a homeless project (ES, SH, SO, TH) and not
-    # enrolled in PH with a move-in date in the past, on the day the process is run.
+    # Clients are included in the submission if:
+    # 1. they are enrolled in a homeless project (ES Entry/Exit, ES NbN, SH, SO, TH)
+    # 2. they have at least one service in the enrollment
+    # 3. they are NOT enrolled in PH with a move-in date in the past, on the day the process is run.
+    # 4. they have at least one entry date in some project in the past 12 months
+    # 5. they have never been exited to a deceased destination
     private def generate_submission
       file_path = File.join(@file_path, submission_filename)
       count = 0
       return send_test_file(file_path) if test_file
 
       seen_medicaid_ids = Set.new
+      clients_with_entries_in_past_year = GrdaWarehouse::ServiceHistoryEnrollment.entry.
+        where(entry_date: 1.year.ago.to_date..)
+      # `homeless_on_date` encompasses ongoing homeless enrollment with with service and no overlapping moved-in PH
       GrdaWarehouse::Hud::Client.homeless_on_date.
```

Comment:
Assign this to a variable? I had trouble following all this logic only to end up with an iteration